pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -147,16 +169,45 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 		if valicationWebhookController != nil {
 			go valicationWebhookController.Start(stopCh)
 		}
+		//add config controller
+		remoteConfigController, err := m.createRemoteConfigController(clusterID, config, opts)
+		if err != nil {
+			return fmt.Errorf("failed to create remote Config controller for cluster %v: %v", clusterID, err)
```

Comment:
Anything user can do in this error?  check the remote secret is correct or contact mesh admin?

Reply:
will add message to let user check their remote secet